app/Resources/views/components/paginators/category.html.twig

```diff
@@ -0,0 +1,21 @@
+{% if paginator is defined %}
```

Comment:
Určitě bych nedělal dva soubory s prakticky stejným kódem... Zkus schválně vymyslet jak na to, pobavíme se o tom na lekci